src/main.js

```diff
@@ -253,11 +253,13 @@ class MiniGraphCard extends LitElement {
 
   renderStates() {
     const { entity, value } = this.tooltip;
-    const state = value !== undefined ? value : (
-      this.config.entities[0].attribute
-        ? this.entity[0].attributes[this.config.entities[0].attribute]
-        : this.entity[0].state
-    );
+    const state = value !== undefined ? value : (this.config.show.state === 'last'
+      ? this.points[0][this.points[0].length - 1][V]
+      : (
+        this.config.entities[0].attribute
+          ? this.entity[0].attributes[this.config.entities[0].attribute]
+          : this.entity[0].state
+      ));
```

Comment:
This is getting quite complex to read.
I would prefer if we could avoid these nested tenary operators by replacing with simpler `if` / `else` statements or similar.

Reply:
Considering a refactor _soon_ to allow most (sensible) configuration items (like value_factor) to be configurable both on root level (for all entities) or on a specific entity; which should also improve readability of this blob (converted to if/else statements _for now_ )